### ISSUE TITLE
Remove supportsRtl and allowBackup from manifest

### ISSUE THREAD

**Issue by PaulWoitaschek** [seiginonakama/BlockCanaryEx#5]
When I want to setup the library I get a build error:

```
Error:
	Attribute application@allowBackup value=(false) from AndroidManifest.xml:40:9-36
	is also present at [com.letv.sarrsdesktop:BlockCanaryExJRT:0.9.5.2] AndroidManifest.xml:12:9-35 value=(true).
	Suggestion: add 'tools:replace="android:allowBackup"' to <application> element at AndroidManifest.xml:38:5-157:19 to override.

Error:
	Attribute application@supportsRtl value=(false) from AndroidManifest.xml:44:9-36
	is also present at [com.letv.sarrsdesktop:BlockCanaryExJRT:0.9.5.2] AndroidManifest.xml:13:9-35 value=(true).
	Suggestion: add 'tools:replace="android:supportsRtl"' to <application> element at AndroidManifest.xml:38:5-157:19 to override.
```

I think a library should not specify these attributes.

**Comment by seiginonakama** [comment]
you are right, the pull request be merged just now, you are welcome.

**Comment by seiginonakama** [comment]
the issue fixed in 0.9.5.3
